src/fairseq2/models/s2t_transformer/builder.py

```diff
@@ -380,6 +388,7 @@ def build_conformer_block(self) -> TransformerEncoderLayer:
         conv = ConformerConvolution(
             self.config.model_dim,
             self.config.depthwise_conv_kernel_size,
+            norm_type=self.config.conv_norm_type,
```

Comment:
As I mentioned above, S2T Transformer never had an architecture using "layer_norm" Conformer blocks. We can remove this "conv_norm_type" parameter from its configuration set.